script.js

```diff
@@ -1,39 +1,60 @@
-let booksList = [];
-const form = document.getElementById('add-form');
-const bookList = document.querySelector('.book-list');
-
-function addForm() {
-  const title = document.getElementById('title');
-  const author = document.getElementById('author');
-  booksList.push({ title: title.value, author: author.value });
-  window.localStorage.setItem('books', JSON.stringify(booksList));
-}
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-form.addEventListener('submit', addForm);
+  static getBooks() {
```

Comment:
- [ ] Because you declared this method as static, your code is not actually working. Right now, it doesn't show anything on the screen, and if you check your console, you will see an error saying that `this.getBooks` is not a function. This happens because of the static declaration of your function. In order to make it work, you will need to:
1. Either remove the static keyword. This would make everything work as expected.
2. Either change all the references from `this.getBooks` to `Book.getBooks`.

Both solutions worked for me, so you are free to choose the one you think fits better, or if you are able to find any other solution, you can go for it as well!